dashboard/src/components/DeploymentForm/BasicDeploymentForm/DiskSizeParam.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from "react";
+import { IBasicFormParam } from "shared/types";
+import Slider from "../../../components/Slider";
+
+export interface IDiskSizeParamProps {
+  id: string;
+  name: string;
+  label: string;
+  param: IBasicFormParam;
+  handleBasicFormParamChange: (
+    name: string,
+    p: IBasicFormParam,
+  ) => (e: React.FormEvent<HTMLInputElement>) => void;
+}
+
+export interface IDiskSizeParamState {
+  Gi: number;
+}
+
+function toNumber(value: string) {
+  // Force to return a Number from a string removing any character that is not a digit
+  return Number(value.replace(/[^\d]/g, ""));
```

Comment:
So if I enter "125.5"? Currently will this set the state to 1255 Gi?

Reply:
True, I thought that decimals were not allowed (but they are according to https://github.com/kubernetes/community/blob/master/contributors/design-proposals/scheduling/resources.md#resource-quantities) so I will add a handle for that.

Reply:
Caveat: Since the minimal step in the slide is 1 unit, copying `125.5` will result in `126` since the slider round it to the closest int.